embedded/store/immustore.go

```diff
@@ -408,7 +408,7 @@ func OpenWith(path string, vLogs []appendable.Appendable, txLog, cLog appendable
 		WithFileMode(opts.FileMode).
 		WithLog(opts.log).
 		WithFileSize(fileSize).
-		WithSynced(false). // index is built from derived data
+		WithSynced(opts.Synced). // built from derived data, but temporarily to reduce chances of data inconsistencies
```

Comment:
I don't fully understand the comment, shouldn't be something like:

```suggestion
		WithSynced(opts.Synced). // built from derived data, but temporarily modified to reduce chances of data inconsistencies until a better solution is implemented
```